precompiles/staking/StakingI.sol

```diff
@@ -149,7 +149,7 @@ interface StakingI is authorization.AuthorizationI {
     ) external returns (bool success);
 
     /// @dev Defines a method for edit a validator.
-    /// @param description Description parameter to be updated, an empty string ""
+    /// @param description Description parameter to be updated, an string "[do-not-modify]"
```

Comment:
Hey @luchenqun, I'm very happy to see you around again! Thanks for adding the fixes, howerver, this description is still not correct because the string should be casted to the proper Description type. The "[do-not-modify]" can be the field value of every Description field, not the value of description.